### PR TITLE
Added the ability to define an afpd server line

### PR DESCRIPTION
**Pull request by tavisto**
When I went to import this into a fresh module repo it complained about not having the puppetlabs-stdlib. 
I also needed the ability to define a custom server line so I added a that functionality, complete with tests.

### CODE REVIEW

**Comment by ryanycoleman** [comment]
Hi @tavisto! Thanks for the contribution! I'm going to make a couple of comments in-line to the code.

File: Modulefile [hunk 1]
```diff
@@ -9,3 +9,4 @@ project_page ''
 
 ## Add dependencies, if any:
 dependency 'ripienaar/concat', '>= 0.1.0'
+dependency 'puppetlabs/stdlib', '>= 2.2.1'
```
Comment (ryanycoleman) [inline Modulefile:12]: Good catch, thank you!

File: manifests/server.pp [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+  validate_string($server_name)
+  if $options != undef {
+      validate_array($options)
+      $options_string = join($options, ' -')
+  }
+
+  concat::fragment { $name:
+    target  => $netatalk::params::afpd_config,
+    content => "${server_name} ${options_string}",
```
Comment (ryanycoleman) [inline manifests/server.pp]: The idea is to be able to declare multiple netatalk::server resources, right? If so, you'll need a newline character at the end of your content parameters value. 

content => "${server_name} ${options_string}\n",

Comment (tavisto) [inline manifests/server.pp]: Ah, I thought the concat module did that automatically. I will update this, also the volumes section didn't have a newline so I didn't add one here.

**Comment by ryanycoleman** [comment]
Thanks for your contributions @tavisto! I'll get a release tagged, documented and pushed up to the Puppet Forge soon.

**Pull request merged**